.circleci/config.yml

```diff
@@ -291,43 +356,55 @@ jobs:
 
 workflows:
   version: 2.1
-  test-publish:
+  test:
```

Comment:
Can you add a comment for the purpose of each `job` and `workflow` and add a comment for what should trigger each workflow?